Source/JavaScriptCore/runtime/StringPrototypeInlines.h

```diff
@@ -69,6 +69,7 @@ ALWAYS_INLINE std::tuple<int32_t, int32_t> extractSliceOffsets(int32_t length, i
 }
 
 template<typename NumberType>
+requires std::integral<NumberType> || std::floating_point<NumberType>
```

Comment:
I think it would be better to use a concept here rather than `requires`. Not sure where to put the concept, though. Right here would be OK for the moment, I guess, or `StdLibExtras.h`.
```suggestion
template<typename T> concept Arithmetic = std::is_arithmetic_v<T>;

template<Arithmetic NumberType>
```

Reply:
Updated